### PR TITLE
Added Contributor

### LINKED ISSUE #108: Adding Contributor section in readme  ( Automatically )
I will like to add a feature this will update contributor list in read me automatically !! Will start to work on this issue as soon as i get assigend !!

### PR DESCRIPTION
**Pull request by rohank2502**
Fixes #108 

#### Changes done
Added Contributor in read me which will be updated automatically 

- [ ]

Screenshots of the changes (If any) -

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

#### ✅️ By submitting this PR, I have verified the following

- [x] Worked on forked repo and starred the main repo
- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [ ] Tried Squashing the commits into one

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,3 +143,18 @@ if (thisRepo.isAwesome()) {
 We got accepted for the GSSoc 2021 and activelty looking for new participants,please feel free to ask questions in the discussion section of this repository and if you want to be our mentor please mention in the discussion.
 
 ### Go to the [Wiki](https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/wiki) for more information.
+
+
+## Contributors:
+
+### Credits goes to these people:✨
+
```
Comment (ayan-biswas0412) [inline README.md:151]:
```suggestion
```
remove unnecessary blank lines

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rohank2502) [reply README.md:151]: Ok !!

Comment (rohank2502) [reply README.md:150]: I have removed unnecessary lines but removing more will make the code unclean and it don't impact the spaces in readme

Comment (ayan-biswas0412) [reply README.md:150]: okk

**Review by ayan-biswas0412 — APPROVED** [review]
nice work

**Pull request merged**